activerecord/lib/active_record/reflection.rb

```diff
@@ -278,6 +277,14 @@ def autosave=(autosave)
         end
       end
 
+      def allow_autosave?
```

Comment:
Was going to use this in a refactor but... I'll leave that up to another PR. Better to keep this one strictly a bugfix.